src/service-implementations/lf-repo-tree-node-service.ts

```diff
@@ -125,10 +124,17 @@ export class LfRepoTreeNodeService implements LfTreeNodeService {
       const parentPath = this.getParentPath(treeNode.path);
       if (parentPath) {
         const parentEntry: FindEntryResult = await this.repoClient.entriesClient.getEntryByPath({ repoId, fullPath: parentPath });
-        if (parentEntry.entry) {
+        if (parentEntry.entry && parentEntry.entry.entryType === EntryType.Folder) {
           const foundParentEntry = parentEntry.entry;
-          foundParentEntry.fullPath = parentPath;
-          return this.createLfRepoTreeNode(foundParentEntry, repoName);
+          if (foundParentEntry.id === 1) {
+          return this.getRootTreeNodeAsync();
+          }
+          const icon: string = this.getIconForEntry(foundParentEntry.entryType);
+          const parentNode = this.createFolderNode(foundParentEntry.name, parentPath, foundParentEntry.id, foundParentEntry.entryType, icon);
+          // this.setNodeProperties(parentNode, foundParentEntry);
+          this.setColumnAttributesForEntry(foundParentEntry, parentNode);
+          this.setTargetIdAndTypeForShortcut(foundParentEntry, treeNode);
```

Comment:
should have an if for is shortcut?

Reply:
it should be handled by setTargetIdAndTypeForShortcut, and getIconsForEntry